projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.scss

```diff
@@ -1,21 +1,69 @@
-.container{
+.open-in-app-container {
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  align-items: center;
-}
-
-.mat-dialog-container{
-  padding: 5px;
-  border-radius: 36px;
-  background-color: var(--theme-blue-button-color);
+  place-content: center;
+  padding: 0;
+  border-radius: 8px;
+  background-color: var(--theme-primary-color);
   color: var(--theme-about-panel-background-color);
+  margin-top: 12px;
 }
 
 .open-in-app.mat-button {
   overflow-x: hidden;
+  font-size: 16px;
+  width: 100%;
+  padding: 0;
+  height: 48px;
 }
 
 .open-in-app.mat-button.cdk-program-focused .mat-button-focus-overlay {
   opacity: 0;
 }
+
+.open-in-app.mat-button.cdk-touch-focused {
+  background-color: var(--theme-primary-color);
+}
+
+.cross-button.mat-button.cdk-program-focused .mat-button-focus-overlay {
+  opacity: 0;
+}
+
+.cross-button.mat-button:focus-visible {
```

Comment:
".mat.button" not needed here, also merge with another ".cross-button" you declared already

Reply:
merged with cross-button already declared.